pokedex/helper.py

```
@@ -38,3 +43,12 @@ def register_subscriber(email):
     wrapper = ConnectionWrapper()
     wrapper.register_subscriber(email)
     pass
+
+
+def fetch_pokemon(id):
+    wrapper = ConnectionWrapper()
+    result = wrapper.get_single_pokemon(id)
+    if result is None:
+        message = "No existing pokemon with id=%s".format(id)
+        ValueError(message)
```

Please raise this exception.